challenges/classes.js

```diff
@@ -1,7 +1,47 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+class NewCuboidMaker {
+  constructor(attrs) {
+    this.length = attrs.length;
+    this.width = attrs.width;
+    this.height = attrs.height;
+  }
+  volume() {
+    return this.length * this.width * this.height;
+  };
+
+  surfaceArea() {
+    return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+  };
+}
+
+const newCuboid = new NewCuboidMaker({
+  length: 4,
+  width: 5,
+  height: 5
+});
+
+class CubeMaker extends CuboidMaker {
+  constructor(cubeAttrs) {
+    super(cubeAttrs);
+  }
+
+  cubeVolume() {
```

Comment:
Clean reuse of the `length` property here, no need to create a new one. Well done!